packages/forms-web-app/__tests__/unit/controllers/examination/check-submission-item/utils/summary-list-item/evidence-or-comment.test.js

```diff
@@ -0,0 +1,67 @@
+const {
+	getSummaryListItemEvidenceOrComment
+} = require('../../../../../../../src/controllers/examination/check-submission-item/utils/summary-list-item/evidence-or-comment');
+const {
+	getSummaryListItem
+} = require('../../../../../../../src/controllers/utils/get-summary-list-item');
+
+describe('controllers/examination/check-submission-item/utils/summary-list-item/evidence-or-comment', () => {
+	describe('#getSummaryListItemEvidenceOrComment', () => {
+		describe('When invoking the getSummaryListItemEvidenceOrComment function', () => {
```

Comment:
same as stated elsewhere. The above describe is telling us the function name. This when should be telling us what is purpose sort of is 

'When a getting the summary list for evidence or comment page'